src/actions/copy-environment/index.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import * as core from '@actions/core';
+import os = require('os');
```

Comment:
please fix the lint warnings; it's a bug in the build systems that those don't fail the build

Reply:
Done.